db/migrate/20180403031030_add_shown_verification_failed_modal_to_channels.rb

```diff
@@ -0,0 +1,6 @@
+class AddShownVerificationFailedModalToChannels < ActiveRecord::Migration[5.0]
+  def change
+    add_column :channels, :shown_verification_failed_modal, :boolean, default: false
```

Comment:
minor: consider adding a `null: false` constraint

Also this manual failure case may only apply to Site channels, so it could be cleaner to move these two flags onto [SiteChannelDetails](https://github.com/brave-intl/publishers/blob/staging/app/models/site_channel_details.rb).